data-egress.tf

```diff
@@ -231,6 +231,26 @@ resource "aws_dynamodb_table_item" "housing_SAS_data_egress_config" {
   ITEM
 }
 
+resource "aws_dynamodb_table_item" "data_warehouse_config" {
+  table_name = aws_dynamodb_table.data_egress.name
+  hash_key   = aws_dynamodb_table.data_egress.hash_key
+  range_key  = aws_dynamodb_table.data_egress.range_key
+
+  item = <<ITEM
+  {
+    "source_prefix":                {"S":    "dwh/zip/*"},
+    "pipeline_name":                {"S":    "DWh-Transform-Json"},
+    "recipient_name":               {"S":    "DataWarehouse"},
```

Comment:
if you look at the other SFT egress entries - this is a team, not just DataWarehouse - I'm not sure if that is an issue, just thought it was worth discussing.